src/Cache/Engine/MemcachedEngine.php

```diff
@@ -145,7 +145,25 @@ public function init(array $config = []): bool
         }
         $this->_setOptions();
 
-        if (count($this->_Memcached->getServerList())) {
+        $serverList = $this->_Memcached->getServerList();
+        if (count($serverList)) {
+            if ($this->_config['persistent'] !== false) {
```

Comment:
```suggestion
            if ($this->_config['persistent']) {
```
null shouldn't be persistent, right?

Reply:
Can we use `$this->_Memcached->isPersistent()` here?

Reply:
We might be able to.